src/components/HeroSection/HeroSection.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import CTAButton from '../CTAButton/CTAButton';
+import {
+  HeroSectionWrapper,
+  HeroSectionTitle,
+  HeroSectionP,
+  HeroSectionButtonLinkWrapper,
+  Table,
+  TableContainer,
+} from './HeroSection.styles';
+
+const HeroSection = () => (
+  <HeroSectionWrapper>
+    <HeroSectionTitle>No reactions to your reddit posts?</HeroSectionTitle>
```

Comment:
I don't think there's a need to prefix all of these components with `HeroSection`. They are already scoped to the HeroSection IMO.